modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/jaxrs/exception/mapper/DefinitionOptionValueRelSkuExceptionMapper.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.headless.commerce.admin.catalog.internal.jaxrs.exception.mapper;
+
+import com.liferay.commerce.product.exception.CPDefinitionOptionValueRelCPInstanceException;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.BaseExceptionMapper;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.Problem;
+
+import javax.ws.rs.ext.ExceptionMapper;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Joao Cordeiro
+ */
+@Component(
+	property = {
+		"osgi.jaxrs.application.select=(osgi.jaxrs.name=Liferay.Headless.Commerce.Admin.Catalog)",
+		"osgi.jaxrs.extension=true",
+		"osgi.jaxrs.name=Liferay.Headless.Commerce.Admin.Catalog.DefinitionOptionValueRelSkuExceptionMapper"
+	},
+	service = ExceptionMapper.class
+)
+public class DefinitionOptionValueRelSkuExceptionMapper
+	extends BaseExceptionMapper<CPDefinitionOptionValueRelCPInstanceException> {
+
+	@Override
+	protected Problem getProblem(
+		CPDefinitionOptionValueRelCPInstanceException
+			cpDefinitionOptionValueRelCPInstanceException) {
+
+		return new Problem(cpDefinitionOptionValueRelCPInstanceException);
```

Comment:
https://liferay.atlassian.net/browse/LPS-181443

Please avoid the `exception.getMessage()` approach as it may contain sensible information. Use the string directly in the exception mapper instead.